strapi/strapi
# [I18n] init create locale modal

### What does it do?

- Initialize the ModalCreate for a locale
- Add the `useDefaultLocales` hook + reducer
- ⚠️ The hook is not used in this PR, it will be done and tested in another one

### Why is it needed?

Add the "add locale" functionality

###  How to test?

Make sure to have the permissions set by modifying `canRead` by `true` in https://github.com/strapi/strapi/blob/features/i18n-front/packages/strapi-plugin-i18n/admin/src/containers/SettingsPage/index.js#L25 

- Log into your account
- Select "Settings" on the left side menu bar
- Select "Internationalization" in the second menu opened
- Click the blue-button with a "+" on the top right corner
- You should see the "add locale" displayed on the screen
